<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>audio_video_properties</class>
 <widget class="QDialog" name="audio_video_properties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Properties</string>
  </property>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filename">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>22</y>
     <width>129</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>22</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Filename:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="duration">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>101</y>
     <width>129</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>101</y>
     <width>63</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>141</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Size:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="size">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>141</y>
     <width>129</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>61</y>
     <width>35</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Path:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="path">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>61</y>
     <width>129</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>181</y>
     <width>58</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Filetype:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filetype">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>181</y>
     <width>129</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
